src/test/java/uk/gov/hmcts/reform/blobrouter/tasks/BlobCleanerTaskTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.blobrouter.tasks;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.blobrouter.config.ServiceConfiguration;
+import uk.gov.hmcts.reform.blobrouter.tasks.processors.ContainerCleaner;
+
+import static java.util.Arrays.asList;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+class BlobCleanerTaskTest {
+    @Mock
+    private ContainerCleaner containerCleaner;
+
+    private BlobCleanerTask task;
+
+    void setUp(ServiceConfiguration serviceConfiguration) {
+        task = new BlobCleanerTask(containerCleaner, serviceConfiguration);
+    }
+
+    @DisplayName("Get all configured containers and process only 1 available container")
+    @Test
+    void should_get_all_containers_but_process_only_available_ones() {
+        // given
+        ServiceConfiguration serviceConfiguration = new ServiceConfiguration();
+        serviceConfiguration.setStorageConfig(
+            asList(
+                configure("bulkscan", true),
+                configure("disabled-service", false)
+            )
+        );
```

Comment:
We could extract this bit into a separate method and reuse that method across tests. Not so important.

Reply:
Will do